lib/test.js

```diff
@@ -134,7 +138,11 @@ Test.prototype.run = function () {
 	}.bind(this));
 };
 
-Test.prototype.end = function () {
+Test.prototype.end = function (err) {
+	if (arguments.length >= 1 && Boolean(err)) {
```

Comment:
Originally this was `!!err` but linter told me to `Boolean(err)` so I followed blindly